dotnet/src/Planners/Planners.Handlebars/Handlebars/HandlebarsPlanner.cs

```diff
@@ -75,19 +78,22 @@ public Task<HandlebarsPlan> CreatePlanAsync(Kernel kernel, string goal, Cancella
 
     private async Task<HandlebarsPlan> CreatePlanCoreAsync(Kernel kernel, string goal, CancellationToken cancellationToken = default)
     {
+        var executionSettings = this._plannerOptions.ExecutionSettings ?? s_defaultPromptExecutionSettings;
+
         // Get CreatePlan prompt template
-        var availableFunctions = this.GetAvailableFunctionsManual(kernel, out var complexParameterTypes, out var complexParameterSchemas);
-        var createPlanPrompt = await this.GetHandlebarsTemplateAsync(kernel, goal, availableFunctions, complexParameterTypes, complexParameterSchemas, cancellationToken).ConfigureAwait(false);
+        var functionsManual = await kernel.Plugins.GetJsonSchemaFunctionsManualAsync(this._plannerOptions, null, null /*logger*/, true, cancellationToken).ConfigureAwait(false);
```

Comment:
Can you share a screenshot of what the function manual in the prompt looks like now?

Reply:
![image](https://github.com/microsoft/semantic-kernel/assets/51341242/85323ddf-ff9f-451a-8f12-bc050ec48097)

![image](https://github.com/microsoft/semantic-kernel/assets/51341242/06d30aa5-859e-40a8-91ab-33f8a7bbd022)